omniledger/service/api.go

```diff
@@ -27,6 +28,7 @@ type Client struct {
 }
 
 // NewClient instantiates a new Omniledger client.
+// TODO: this needs to be changed to avoid having an invalid Client.
```

Comment:
I wouldn't understand how to fix this todo.

Reply:
Changing the signature of the method to `NewClient(roster, ...)`. Now if you call `NewClient()`, you get a client that is not usable... Which _might_ be OK, but we get weird panics out of it.